_posts/2024-06-20-pyconus-pyopensci-sprints.md

```diff
@@ -0,0 +1,358 @@
+  <figcaption> We like to label issues as help-wanted and sprintable. A sprintable issue is a discrete issue that could be tackled (and ideally completed) during a sprint. A sprint's duration can span from  a half day to multiple full days depending on the event. What is most important is that the issue is confined in some way so a single person could tackle it in a reasonable amount of time.</figcaption>
+</figure>
+
+You never know when a contributor might be looking for a data science project to
+work on. It's always helpful totag issues that could be completed or started
+during a sprint as `sprintable` and `help-wanted` if you think someone outside
+of your team could tackle the issue. This tip came from
+[Inessa Pawson](https://github.com/InessaPawson), who has extensive experience
+with contributors through her work on the
+[Contributor Experience Project](https://contributor-experience.org/).
+
+### Collect issues in a single place - the help-wanted board
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/june/pyopensci-sprint-project-board.png" alt="A graphic that shows the pyOpenSci GitHub project board. The graphic shows two columns of the project board. The first is Python Packaging and it has 7 issues in that column. The second column says Beginner-friendly / non technical. The point of this graphic is just to show a user that a project board is a nice way to organize issues that contributors might want to challenge during a sprint. At the bottom there is text that says includes a beginner-friendly column and should have easy-to-understand issue titles." style="width: 80%;" />
+  <figcaption> Project boards help you organize your sprintable tasks into categories that might help a contributor find something that interests them. Here we have a beginner-friendly column for beginners. but we also include columns that are topical such as Python programming, ci / devops and Sphinx. Also notice that the issues are generally clearly titled.</figcaption>
+</figure>
+
+Curate a `help-wanted` board that contains all the issues that could be completed
+by an outside community member in one place.
+
+Our pyOpenSci `help-wanted` board is a
+[GitHub project board](https://github.com/orgs/pyOpenSci/projects/3/views/2)
+that allows us to organize issues that we think others could work on by type,
+including:
+
+* Beginner-friendly
+* Python
+* DevOps / CI
+
+### Infrastructure and automation of issues
+
+PyOpenSci has many GitHub repositories where people might label an issue as
+`help-wanted` and forget to add it to our sprint help-wanted project board. To
+address this, we set up
+a [CI workflow](https://github.com/pyOpenSci/pyopensci.github.io/blob/main/.github/workflows/add-help-wanted.yml)
+that can be added to any repository. This workflow moves any issue labeled
+`sprintable` or `help-wanted` to our help-wanted GitHub project board.
+
+**GitHub project boards support project workflows** that auto-add issues to a
```

Comment:
ADD: You can also setup a workflow to remove /archive issues from the board when they are closed...